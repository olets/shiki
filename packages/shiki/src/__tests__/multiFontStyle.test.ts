import { getHighlighter } from '../index'

test('Handle multiple font styles', async () => {
  const highlighter = await getHighlighter({
    theme: 'material-theme',
    langs: ['md']
  })
  const out = highlighter.codeToHtml(`***bold italic***`, { lang: 'md' })
  expect(out).toMatchSnapshot()
})
