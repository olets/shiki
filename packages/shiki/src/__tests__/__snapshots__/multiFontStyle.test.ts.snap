// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Handle multiple font styles 1`] = `"<pre class=\\"shiki material-theme\\" style=\\"background-color: #263238\\" tabindex=\\"0\\"><code><span class=\\"line\\"><span style=\\"color: #89DDFF; font-weight: bold\\">**</span><span style=\\"color: #89DDFF; font-style: italic; font-weight: bold\\">*</span><span style=\\"color: #F07178; font-style: italic; font-weight: bold\\">bold italic</span><span style=\\"color: #89DDFF; font-style: italic; font-weight: bold\\">*</span><span style=\\"color: #89DDFF; font-weight: bold\\">**</span></span></code></pre>"`;
